runtime/doc/options.txt

```diff
@@ -1525,9 +1531,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 		If the Dict returned by the {func} includes {"refresh": "always"},
 		the function will be invoked again whenever the leading text
 		changes.
-		If generating matches is potentially slow, |complete_check()|
-		should be used to avoid blocking and preserve editor
-		responsiveness.
+		If generating matches is potentially slow, call
+		|complete_check()| periodically to keep Vim responsive. This
```

Comment:
Is this like "yield"? I wonder why it is specific to completion, instead of just `yield()` or something like that.

Reply:
All docs I could find via `:helpgrep complete_check` describe what sounds exactly like `os_breakcheck`, i.e. a pseudo "yield" that checks for input.

Reply:
Hmm, it seems that `complete_check()` also updates the popup menu.